ui/new-public-messages.js

```diff
@@ -36,8 +44,26 @@ module.exports = function (state) {
           live(),
           toPullStream(),
           pull.drain((msg) => {
-            if (!msg.value.meta)
+            if (!msg.value.meta) {
               self.newPublicMessages = true
+
+              // If we're scrolled to the top of the page and autorefresh is on, refresh.
+              if (self.publicRefreshTimer == 0) {
```

Comment:
this needs to be only enabled if the settings is true. We don't want to do anything unless that is set.

Reply:
That's checked inside of refreshIfConfigured.  It's checked there because new-public-messages is instantiated once and doesn't get reloaded as you change between tabs, so if you turn off the setting and save, the timer might still autorefresh once if it was checked outside of refreshIfConfigured.

Reply:
Right, but I don't want to to do anything if not configured. Will be much easier to figure out if its working or not.

Reply:
See latest commit.